core/jazz_ui/src/config/toastmessages.ts

```diff
@@ -95,7 +95,15 @@ export const toastMessage = {
         "error400":"Your registration failed. Seems like something went wrong.",
         "error401":"Token expired",
         "error404":"Your registration failed. Seems like something went wrong.",
-        "error500":"Your registration failed. Seems like something went wrong."},
+        "error500":"Your registration failed. Seems like something went wrong."
+    },
+    "resetPwd":{
+        "success":"A verification code is sent to your registered email.",
+        "error400":"Cannot reset password for the user as there is no registered/verified email or phone_number.",
```

Comment:
Remove "phone_number". As of now the registration and verification happen only through email.

Reply:
Updated.